packages/ses/test/import.test.js

```diff
@@ -4,10 +4,264 @@
 /* eslint max-lines: 0 */
 
 import test from 'ava';
+import { StaticModuleRecord } from '@endo/static-module-record';
 import '../index.js';
 import { resolveNode, makeNodeImporter } from './node.js';
 import { makeImporter, makeStaticRetriever } from './import-commons.js';
 
+test('module map primed with module source', async t => {
```

Comment:
The individual commits are framed to show which of these tests were failing before this refactor.